src/modules/discovery/kubelet.py

```diff
@@ -53,6 +55,7 @@ def get_secure_access(self):
             self.publish_event(SecureKubeletEvent(secure=True, anonymous_auth=False))
 
     def ping_kubelet(self):
+        logging.debug("Attempting to ping kubelet")
```

Comment:
I know the function is called ping_kubelet but I think that's a bit misleading ("ping" to me implies ICMP rather than HTTP). Maybe "Attempting to get pod info from kubelet"?